src/main/java/com/schematica/world/schematic/SchematicAlpha.java

```diff
@@ -33,6 +33,7 @@
     private static final String OFFSET_Y = "OffsetY";
     private static final String OFFSET_Z = "OffsetZ";
 
+    @SuppressWarnings("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)